src/main/java/io/appium/java_client/service/local/AppiumServiceBuilder.java

```diff
@@ -36,14 +36,36 @@
 
 public final class AppiumServiceBuilder extends DriverService.Builder<AppiumDriverLocalService, AppiumServiceBuilder> {
 
+    /**
+     * The property of environmental variable used to define
+     * the path to executable appium.js (node server v<=1.4.x) or
+     * main.js (node server v>=1.5.x)
+     */
     public static final String APPIUM_PATH = "APPIUM_BINARY_PATH";
+
+    /**
+     * The property of environmental variable used to define
+     * the path to executable NodeJS file (node.exe for WIN and
+     * node for Linux/MacOS X)
+     */
     public static final String NODE_PATH = "NODE_BINARY_PATH";
 
     private static final String APPIUM_FOLDER = "appium";
+
     private static final String BIN_FOLDER = "bin";
+    private static final String BUILD_FOLDER = "build";
+    private static final String LIB_FOLDER = "lib";
+
     private static final String APPIUM_JS = "appium.js";
-    private static final String APPIUM_NODE_MASK =  File.separator +
-            APPIUM_FOLDER + File.separator + BIN_FOLDER + File.separator + APPIUM_JS;
+    private static final String MAIN_JS = "main.js";
+
+    private static final String ERROR_TEXT_WHEN_DEFAULT_APPIUM_NODE_IS_NOT_FOUND = "There is no installed nodes! Please " +
```

Comment:
`ERROR_TEXT_WHEN_DEFAULT_APPIUM_NODE_IS_NOT_FOUND` is too long. How about:

`ERROR_NODE_NOT_FOUND`